### DISCUSSION TITLE
[Checkout] Problem with Marketplace + Seller + Buyer

### DISCUSSION THREAD

**Discussion by tomasmalio** [mercadopago/sdk-js#78] [Q&A] [answered]
Hi everyone,
How are they? I'm having a problem at checkout with a marketplace and seller configuration.
In the last step, it throws me an "Oh no, something went wrong" error. I don't understand what is happening because afterwards it allows me to do everything.

<img width="1512" alt="Captura de Pantalla 2022-07-12 a las 17 50 45" src="https://user-images.githubusercontent.com/1799432/178756935-2e6de6dd-ee96-4c2d-bcec-ef20081f9709.png">

In the configuration I am adding my marketplace to the body: 'MP-MKT-APP_ID' and the marketplace_fee. On the other hand, in the header sending the seller's ACCESS_TOKEN.

At the time of purchase I am testing with the test cards and the buyer's mail.

I have tried two different ways:
1) Marketplace (original account using test credentials) + Seller (test) + Buyer (test)
2) Marketplace + Seller + Buyer all with test users

Someone could help me? Thank you very much!

**Comment by lucmantovani** [comment, marked as answer]
Hello @tomasmalio!

Could you tell me if in production it works correctly?
It seems that it may be an issue with our tests environment

**Reply by tomasmalio** [reply]
Hi @lucmantovani,
In production it works but we need to test all the app so I don't know what is happening with the sandbox env

**Reply by lucmantovani** [reply]
I'm glad in production works!
We indeed are facing some issues with the sandbox environment... But we are already working to fix it!
As soon as it is fixed I will let you know!

**Reply by carlosoro** [reply]
@lucmantovani do you have an ETA for this fix?

**Reply by lucmantovani** [reply]
Unfortunately not yet... But I'm in direct contact with the responsible team and I will let you know when it is fixed!

**Reply by lucmantovani** [reply]
Hello @tomasmalio and @carlosoro!

I have been notified that the problem was fixed, tried some requests and all of them were successful!
Could you validate as well, please?
